Assets/Code/Scripts/Level generation scripts/FillerGenerationScript.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+
+public class FillerGenerationScript : TileGenerationScript
+{
+
+    public override GameObject Generate(int i, int j)
+    {
+        return Instantiate(tiles[0], new Vector3(i * Width, 0, j * Width), Quaternion.identity);
+    }
```

Comment:
`CrossGenerationScript` and `FillerGenerationScript` are basically the same at the moment, maybe just adding a TODO or a JSDoc block at the top could prevent confusion as to their purpose